event/manager.go

```diff
@@ -158,6 +158,17 @@ func eventChangeHandler(event *api.Event, preEvent *api.Event) {
 			log.Errorf("delete finished event err: %v", err)
 		}
 	}
+
+	// make the state of version as running, the frontend will be based on this state to decide whether to show cancel button.
+	if preEvent.Status == api.EventStatusPending && event.Status == api.EventStatusRunning && event.Operation == CreateVersionOps {
```

Comment:
You can set the status of version to  "api.VersionRunning" when the event create.